### ISSUE TITLE
Comments on notebooks

### ISSUE THREAD

**Issue by nicoguaro** [barbagroup/CFDPython#45]
For openjournals/jose-reviews#21.

**Comment by nicoguaro** [comment]
## 01_Step_1

``ipython notebook`` >> ``jupyter notebook``

**Comment by nicoguaro** [comment]
## 02_Step_2

- Cell [1],
  ``import numpy                 #we're importing numpy and calling it np locally``
  NumPy is called ``numpy`` and not ``np``

**Comment by nicoguaro** [comment]
## 05_Step_4

- Cell [3], I think that it would be more consistent to use ``sympy.pi`` instead of ``numpy.py``
  ``phi = (sympy.exp(-(x - 4 * t)**2 / (4 * nu * (t + 1))) +
       sympy.exp(-(x - 4 * t - 2 * numpy.pi)**2 / (4 * nu * (t + 1))))`` >> ``phi = (sympy.exp(-(x - 4 * t)**2 / (4 * nu * (t + 1))) +
       sympy.exp(-(x - 4 * t - 2 * sympy.pi)**2 / (4 * nu * (t + 1))))``

**Comment by nicoguaro** [comment]
## 06_Array_Operations_with_NumPy

- After Cell [6]: it might be useful to mention that the running time depends on the computer. For some people it might be obvious, but that's not always the case.

**Comment by nicoguaro** [comment]
## 13_Step_10

- Before cell [1], I would change ``b_{i,j}=-100$ at $i=nx*3/4, j=3/4 ny`` for ``b_{i,j}=-100$ at $i=3/4 nx, j=3/4 ny``

**Comment by nicoguaro** [comment]
## 14_Optimizing_Loops_with_Numba

- The notebook mixes spaces and tabs.
- ``laplace2d`` functions mix the alias for NumPy, using ``numpy`` and ``np``. Due to this  the notebook does not work, the following is the returned error

  > NameError: name 'np' is not defined

**Comment by nicoguaro** [comment]
## 15_Step_11

It might be a good idea to consider streamplots besides glyphs for the vector visualization. It suffices to change

    pyplot.quiver(X, Y, u, v)

for

    pyplot.streamplot(X, Y, u, v)

For some vector fields this is easier to understand, and introduces another visualization technique.

This comment is a matter of taste, so I understand if the authors don't consider it needed.

**Comment by nicoguaro** [comment]
## 17_NumbaPro

- The extension for this notebook has a ``.bak`` appended.

**Comment by nicoguaro** [comment]
## 17_NumbaPro

- The extension for this notebook has a ``.bak`` appended.

**Comment by nicoguaro** [comment]
## 18_Burgers_equation

- Cell [6] returns the following error

  > from JSAnimation.IPython_display import display_animation

  JSAnimation is not mentioned as a dependency. A workaround that is compatible with ``FuncAnimation```is to use the IPython magic ``%matplotlib notebook`` instead of ``%matplotlib inline`

**Comment by nicoguaro** [comment]
## 19_Odd_Even_Decoupling

The notebook depends on a folder with videos that is missing from the repository.

**Comment by gforsyth** [comment]
Notebooks 17, 18 and 19 were never really edited or tested -- at this point my inclination is to just delete them (or move them to a subdirectory).  Especially since numbapro no longer exists and the numba stuff in general is all very out of date.

Thoughts, @labarba ?

**Comment by gforsyth** [comment]
@labarba -- I've pushed up individual commits addressing the comments on notebooks 1, 2, 5, 6 and 13, respectively, in #46 .  If you disagree with any of the fixes just let me know and I can pull out the individual commits in question.

I like the suggestion about using `streamplot` (which, if I recall, was less reliable back when we wrote these).  

Here's the existing quiver plot:
![quiver](https://user-images.githubusercontent.com/3596999/42281425-7df9d92e-7f71-11e8-8cae-6f41d74f34c7.png)
And here's the same plot using `streamplot` instead.
![streamplot](https://user-images.githubusercontent.com/3596999/42281437-860ee9ec-7f71-11e8-918c-903c6e661546.png)

Preference?
